GBZG/View/Home/ThemeDetailView.swift

```diff
@@ -0,0 +1,208 @@
+//
+//  ThemeDetailView.swift
+//  GBZG
+//
+//  Created by KiWoong Hong on 2023/01/18.
+//
+
+import SwiftUI
+
+struct ThemeDetailView: View {
+    let posterSize = UIScreen.main.bounds.size.width
+    
+    let point = 3.5
+    
+    var body: some View {
+        VStack {
+            ScrollView(showsIndicators: false) {
+                visitedThemeImageContainer
+                visitedThemeInfoContainer
+                    .padding(.horizontal, 16)
+            }
+        }
+        .frame(maxWidth: .infinity, maxHeight: .infinity)
+        .toolbar {
+            ToolbarItem {
+                NavigationLink(destination: Text("WriteView")) {
+                    Image("Pencil")
+                }
+            }
+        }
+        .ignoresSafeArea()
+    }
+}
+
+extension ThemeDetailView {
```

Comment:
```suggestion

private extension ThemeDetailView {
```

익스텐션 내에 있는 함수들이 다른 모듈 및 객체에서 사용되거나 호출될 일이 없다면 private으로 접근제어를 지정하는게 어떨까용?
접근제어는 컴파일 시점에 다른 객체에서 사용될지 안될지 알려줌으로써 성능 개선이 된다고 합니다!

Reply:
헉! 반영 했습니다!